pkg/pmem-device-manager/pmd-util.go

```diff
@@ -0,0 +1,56 @@
+package pmdmanager
+
+import (
+	"fmt"
+
+	pmemexec "github.com/intel/pmem-csi/pkg/pmem-exec"
+	"k8s.io/klog/glog"
+	"os"
+	"strconv"
+)
+
+func clearDevice(device PmemDeviceInfo, flush bool) error {
+	glog.Infof("clearDevice: path: %v flush:%v", device.Path, flush)
+	// by default, clear 4 kbytes to avoid recognizing file system by next volume seeing data area
+	var blocks uint64 = 4
+	if flush {
+		// clear all data if "erase all" asked specifically
+		blocks = 0
+	}
+	return flushDevice(device, blocks)
+}
+
+func flushDevice(dev PmemDeviceInfo, blocks uint64) error {
```

Comment:
I would suggest make it public.

Reply:
note that flushDevice was not public already before current change